packages/graphql-server/src/schema.graphql

```diff
@@ -5,9 +5,11 @@
 
 type Query {
   # contact
-  contact(id: String!): Contact!
+  getContact(
+    id: String!,
+  ): Contact!
 
-  contacts(
+  getContacts(
```

Comment:
It should use the name on Query when you fetch swager-to-graphql because it have to consistent with the API term